File: frontend/src/app/dto/quiz-collection/quiz-collection.update.res.dto.ts
import { QuizCollectionUpdateDataResDto } from "./quiz-collection.update.data.res.dto";

export class QuizCollectionUpdateResDto{
    private _data! : QuizCollectionUpdateDataResDto
	private _message! : string

    public get data(): QuizCollectionUpdateDataResDto {
        return this._data;
    }

    public set data(data: QuizCollectionUpdateDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }
    
}

File: src/main/java/com/lawencon/elearn/dto/quizcollection/QuizCollectionUpdateResDto.java
package com.lawencon.elearn.dto.quizcollection;

public class QuizCollectionUpdateResDto {
	private QuizCollectionUpdateDataResDto data;
	private String message;
	public QuizCollectionUpdateDataResDto getData() {
		return data;
	}
	public void setData(QuizCollectionUpdateDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
